programming/navigation/set-actor-parameters.rst

```diff
@@ -0,0 +1,124 @@
+.. _set-actor-parameters:
+
+Setting Actors Parameters
+=========================
+
+The navigation mesh is built based on the parameters set for the actor, or say,
+the capabilities of the actors. The navigation mesh varies to accomodate 
+the size of actor, the climbing capabilities.
+
+Implementation:
+
+First we need to create an object for the class NavMeshBuilder via:
+
+.. only:: python
+
+   .. code-block:: python
+
+      builder = navmeshgen.NavMeshBuilder()
+
+.. only:: cpp
+
+   .. code-block:: cpp
+
+      NavMeshBuilder builder = new NavMeshBuilder();
+
+Now, you can set parameters for a navigation mesh in the following way.
+Remember to set parameters before building the mesh by calling the build() function. 
+
+Actor Height:
+
+The default value of actor height is 2 units. You can set the height of actor to 3.5 units as:
+
+.. only:: python
+
+   .. code-block:: python
+
+      builder.set_actor_height(3.5)
+
+.. only:: cpp
+
+   .. code-block:: cpp
+
+      builder.set_actor_height(3.5);
+
+Actor Radius:
+
+The default value of actor radius is 0.6 units. You can set the radius of actor to 1.0 unit as:
+
+.. only:: python
+
+   .. code-block:: python
+
+      builder.set_actor_radius(1)
+
+.. only:: cpp
+
+   .. code-block:: cpp
+
+      builder.set_actor_radius(1);
+
+Actor Climb:
+
+The default value of actor's climbing capability is 0.9 units. You can set the radius of actor to 1.5 unit as:
+
+.. only:: python
+
+   .. code-block:: python
+
+      builder.set_actor_climb(1.5)
+
+.. only:: cpp
+
+   .. code-block:: cpp
+
+      builder.set_actor_climb(1.5);
+
+You can also get the parameters value as:
```

Comment:
"the parameter's value"

Also, prefer property interface over getters/setters, if we have a property interface